docs/pages/2020_BIGMAP/sections/basics.rst

```diff
@@ -410,7 +411,7 @@ Now, exit the IPython shell and once more check for *all* processes:
 
 .. code-block:: console
 
-    $ verdi process list -a
+    $ verdi process list --all
```

Comment:
I mean...ok but why? `-a` should work fine no?

Reply:
See #295